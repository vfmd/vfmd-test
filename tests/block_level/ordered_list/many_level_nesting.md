Normal text

  1. Level 1
  2. Level 1
    1. Level 2
    2. Level 2

      Level 2
      1. Level 3

        Level 3
      Level 3

           1. Level 4
Level 4
Level 4
               1. Level 5
           2. Level 4
    3. Level 2
      2. Level 3
    4. Level 2
  3. Level 1
Level 1

Normal text
