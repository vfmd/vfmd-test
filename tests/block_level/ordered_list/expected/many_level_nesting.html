<p>Normal text</p>

<ol>
<li>Level 1</li>
<li>Level 1
<ol>
<li>Level 2</li>
<li>Level 2
<p>Level 2</p>

<ol>
<li>Level 3
<p>Level 3
Level 3</p>

<ol>
<li>Level 4
Level 4
Level 4
<ol>
<li>Level 5</li>
</ol></li>
<li>Level 4</li>
</ol></li>
</ol></li>
<li>Level 2
<ol>
<li>Level 3</li>
</ol></li>
<li>Level 2</li>
</ol></li>
<li>Level 1
Level 1</li>
</ol>

<p>Normal text</p>
